build/media_source/system/js/fields/pagebuilder/components/elements/item.vue

```diff
@@ -65,9 +65,10 @@
       ...mapGetters([
         'getType'
       ]),
-      classes() {
+           classes() {
         return [
           'item',
+          `${this.element.key>1 ? '' : 'col-10'}`,        
```

Comment:
Check whitespace error here also, use code sniffer to check these types of error